File: Tareas/Unidad2/a2253332351_practica02/15_ConversionDolarPeso.cpp
#include <iostream>
using namespace std;

int main() {
    double dolares;
    double tipoCambio;
    double pesos;

    cout << "Ingrese la cantidad en dólares: ";
    cin >> dolares;
    cout << "Ingrese el tipo de cambio: ";
    cin >> tipoCambio;

    pesos = dolares * tipoCambio;

    cout << "La cantidad equivalente en pesos es: " << pesos << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica02/15_ConversionDolarPeso.java
import java.util.Scanner;

public class ConversionDolarPeso {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double dolares, tipoCambio, pesos;

         System.out.println("Ingrese la cantidad en dólares: ");
         dolares = entrada.nextDouble();
         System.out.println("Ingrese el tipo de cambio: ");
         tipoCambio = entrada.nextDouble();

         pesos = dolares * tipoCambio;

         System.out.println("La cantidad equivalente en pesos es: " + pesos);
    }
}
